sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -551,9 +559,13 @@ def merge(
       return merged.reset_index(drop=True)
 
   @frame_base.args_to_kwargs(pd.DataFrame)
-  def nlargest(self, **kwargs):
-    if 'keep' in kwargs and kwargs['keep'] != 'all':
+  @frame_base.populate_defaults(pd.DataFrame)
+  def nlargest(self, keep, **kwargs):
+    if keep == 'any':
+      keep = 'first'
```

Comment:
`any` is not part of the pandas API. I assume this is an option you're adding to provide a way to get exactly `n` without promising to get the first or last since that's order-sensitive?

That makes sense - but we should probably document the places where we diverge from pandas like this (could just be a TODO here and/or a jira for now).

Reply:
Also: I take it PartitioningSession revealed this problem with first/last? That's good to know, I was wondering about these options earlier but figured I was misunderstanding something.

Reply:
Yep, exactly. Dropped a TODO. I'm thinking we may want a context (like we have for non-parallel operations) for operations like this to explicitly say that one doesn't care about ordering (e.g. `nlargest` would default to `any` so `df.nlargest(10)` would just work as-is. 


PartitioningSession caused the rows to get perumuted.